volmdlr/faces.py

```diff
@@ -3934,10 +3934,19 @@ def surface2d(self):
         return self._surface2d
 
     def to_dict(self):
-        return {'object_class': 'volmdlr.faces.Triangle3D',
-                'point1': self.point1.to_dict(),
-                'point2': self.point2.to_dict(),
-                'point3': self.point3.to_dict()}
+        dict_ = dc.DessiaObject.base_dict(self)
+        dict_['point1'] = self.point1.to_dict()
+        dict_['point2'] = self.point2.to_dict()
+        dict_['point3'] = self.point3.to_dict()
+        dict_['name'] = self.name
+        dict_['object_class'] = 'volmdlr.faces.Triangle3D'
+        return dict_
```

Comment:
redundant to put both object_class, name and base_dict

Reply:
I deleted  dict_['name'] = self.name and dict_['object_class'] = 'volmdlr.faces.Triangle3D'